code/game/mecha/combat/gygax.dm

```diff
@@ -94,10 +94,6 @@
 */
 
 /obj/mecha/combat/gygax/stopMechWalking()
```

Comment:
you can delete this one too

Reply:
oh ok

Reply:
wait no actually that needs to be there to override the basic stopMechWalking() proc

Reply:
i don't particularly want to dig around in that proc

Reply:
might as well add a return just to avoid leaving it empty i guess. I know it returns by default but :man_shrugging: